docs/custom-rules.md

```diff
@@ -51,20 +51,24 @@ To initialize a custom rule directory, run `python -m detection_rules custom-rul
 ```yaml
 rule_dirs:
   - rules
-  - rules_building_block
 files:
   deprecated_rules: deprecated_rules.json
   packages: packages.yaml
   stack_schema_map: stack-schema-map.yaml
   version_lock: version.lock.json
 directories:
-  actions_dir: actions
-  exceptions_dir: exceptions
+  action_dir: actions
+  exception_dir: exceptions
 ```
 
-* Note: The paths in this file are relative to the custom rules directory (CUSTOM_RULES_DIR/)
-* Note: Refer to each original source file for purpose and proper formatting
-* Note: You can also add an optional `bbr_rules_dirs` section for custom BBR rules.
+Some notes:
+
+* The paths in this file are relative to the custom rules directory (CUSTOM_RULES_DIR/)
+* Refer to each original source file for purpose and proper formatting
```

Comment:
You can probably link to it in this markdown. 
```suggestion
* Refer to each original [source file](../detection_rules/etc/example_test_config.yaml) for purpose and proper formatting
```

Reply:
Perfect, thanks!